aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -992,6 +993,7 @@ pub(crate) struct MockOutput<K, E> {
     pub(crate) read_group_sizes: Vec<(K, u64)>,
     pub(crate) materialized_delta_writes: OnceCell<Vec<(K, WriteOp)>>,
     pub(crate) total_gas: u64,
+    pub(crate) skipped: bool,
```

Comment:
The flag means we are skipping rest of the block?

Reply:
we pad the output with skipped outputs, those have this flag set to true